src/libraries/Common/src/Interop/Windows/Crypt32/Interop.DATA_BLOB.cs

```diff
@@ -31,6 +31,8 @@ internal byte[] ToByteArray()
                 Marshal.Copy(pbData, array, 0, (int)cbData);
                 return array;
             }
+
+            internal unsafe ReadOnlySpan<byte> DangerousAsSpan() => new ReadOnlySpan<byte>((void*)pbData, (int)cbData);
```

Comment:
I threw "Dangerous" in the name to follow the convention of other "Dangerous" APIs that can give you back un-tracked native resources.